SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Microsoft.AppCenter.WindowsDesktop.csproj

```diff
@@ -14,16 +14,17 @@
   <Import Project="..\Microsoft.AppCenter.Shared\Microsoft.AppCenter.Shared.projitems" Label="Shared" />
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
-    <PackageReference Include="sqlite-net-pcl" Version="1.3.1" />
+    <PackageReference Include="SQLitePCLRaw.bundle_green" Version="1.1.14" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
     <PackageReference Include="System.Management" Version="4.6.0" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.6.0" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)'=='net45'">
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Management" />
     <Reference Include="System.Configuration" />
     <Reference Include="System.Deployment" />
+    <Reference Include="System.Management" />
+    <Reference Include="System.Net.Http" />
```

Comment:
Why is System.Net.Http added? Where do we use it?

Reply:
It was there implicitly (broken because of change dependencies). It's used for all network requests on Windows. See `HttpNetworkAdapter`

PS There was a valid point https://github.com/microsoft/appcenter-sdk-dotnet/pull/1234#discussion_r352792084 to verify if we need additional NuGet for it on .NET Core